userpatches/customize-image.sh

```diff
@@ -61,10 +61,12 @@ chage -d 99999999 root
 curl https://install.yunohost.org/stretch | bash -s -- -a
 rm /var/log/yunohost-installation*
 
-# Install InternetCube dependencies
-apt install php7.0-fpm sipcalc hostapd iptables iw dnsmasq # hotspot
-apt install sipcalc dnsutils openvpn curl fake-hwclock # vpnclient
-apt install php-cli php-common php-intl php-json php-mcrypt php-pear php-auth-sasl php-mail-mime php-patchwork-utf8 php-net-smtp php-net-socket php-net-ldap2 php-net-ldap3 php-zip php-gd php-mbstring php-curl # roundcube
+# Install InternetCube dependencies usb detection, hotspot, vpnclient, roundcube
+apt-get install -o Dpkg::Options::='--force-confold' -y --force-yes \
+  file udisks2 udiskie ntfs-3g jq \
+  php7.0-fpm sipcalc hostapd iptables iw dnsmasq firmware-linux-free \
+  sipcalc dnsutils openvpn curl fake-hwclock \
+  php-cli php-common php-intl php-json php-mcrypt php-pear php-auth-sasl php-mail-mime php-patchwork-utf8 php-net-smtp php-net-socket php-net-ldap2 php-net-ldap3 php-zip php-gd php-mbstring php-curl
```

Comment:
So uh I do understand the need for the internetcube (though could be re-checked with people maintaining roundcube_ynh ... some of the deps may be outdated / uneeded these days because deps are installed using composer but they were never removed from install script)

but for Yunohost images I don't think we want to install this by default :s ?

Reply:
Ofcourse, I have done some modification today for adressing this problem, https://github.com/HugoPoi/arm-images/commit/12e159dfc52828368d35eaaa3311849304c42f42

This modification will be included before I merge this PR in a new branch.